openai/openai.go

```diff
@@ -0,0 +1,317 @@
+package openai
+
```

Comment:
Maybe a comment in here saying this is for (partial-)compatibility to openai and that ideally people should use the native REST API for the most functionality/features.